ps_mbo.php

```diff
@@ -95,13 +95,13 @@ class ps_mbo extends Module
     public function __construct()
     {
         $this->name = 'ps_mbo';
-        $this->version = '4.12.0';
+        $this->version = self::VERSION;
```

Comment:
Unfortunately, I believe we can't do that because of a check in addons.

Reply:
Ah ok too bad, I'll hard-code it then